app/main.py

```diff
@@ -1,13 +1,31 @@
-def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+def format_linter_error(
```

Comment:
You need to wrap arguments on a new line if the length of the line with all the arguments does not satisfy the rules of flake8 or PEP8